pkgs/applications/science/molecular-dynamics/lammps/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, writeText, fetchurl,
+  libpng, fftw,
+  mpiSupport ? false, mpi ? null
+}:
+
+assert mpiSupport -> mpi != null;
+
+stdenv.mkDerivation rec {
+  version = "16Feb16";
+  name = "lammps-${version}";
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/project/lammps/${ name }.tar.gz";
+    sha256 = "058459f3053cca0b8c0fdd05cca7acab6ac00934c6da8c9008aa0f55fb5ceefb";
```

Comment:
Super nitpick: I find base32 encoded checksums more visually pleasing (you can convert it with `nix-hash`).